c2corg_ui/templates/image/edit.html

```diff
@@ -1,11 +1,11 @@
 <%!
 from c2corg_common.attributes import default_langs, activities, image_types, image_categories
 %>
-<%inherit file="../base.html"/>
+    <%inherit file="../base.html"/>
```

Comment:
Grmbl. Probably because I'm trying a new editor, vscode instead of the usual atom. I'll correct this (and I should have a better look at my commits to avoid stupid things like this, I'm sorry)